client/linuxabi/linux_abi.go

```diff
@@ -43,14 +45,18 @@ const (
 	iocSnpWithoutNr    = ((iocWrite | iocRead) << iocDirshift) |
 		(iocTypeSnpGuestReq << iocTypeshift) |
 		// unsafe.Sizeof(snpUserGuestRequest)
-		(24 << iocSizeshift)
+		(32 << iocSizeshift)
 
 	// IocSnpGetReport is the ioctl command for getting an attestation report
 	IocSnpGetReport = iocSnpWithoutNr | (0x0 << iocNrshift)
 
 	// IocSnpGetReport is the ioctl command for getting an extended attestation report that includes
 	// certificate information.
 	IocSnpGetExtendedReport = iocSnpWithoutNr | (0x2 << iocNrshift)
+
+	guestMsgVersion = 1
```

Comment:
Can you add a comment explaining what this is?

Reply:
Done.